File: app/app/src/main/java/me/williamhester/reddit/ui/VotableCallbacks.java
package me.williamhester.reddit.ui;

import me.williamhester.reddit.models.Votable;

/** Callbacks for when a votable is clicked. */
public interface VotableCallbacks {
  /** Called when a votable is clicked. */
  void onVotableClicked(Votable votable);
}

File: app/app/src/main/java/me/williamhester/reddit/ui/VotableCallbacks.kt
package me.williamhester.reddit.ui

import me.williamhester.reddit.models.Votable

/** Callbacks for when a votable is clicked.  */
interface VotableCallbacks {
  /** Called when a votable is clicked.  */
  fun onVotableClicked(votable: Votable)
}
